src/EventStore.Core/Services/Storage/ReaderIndex/AllReader.cs

```diff
@@ -311,7 +319,9 @@ public class AllReader<TStreamId> : IAllReader {
 
 		private static bool IsCommitAlike(ILogRecord rec) {
 			return rec.RecordType == LogRecordType.Commit
-			       || ((rec.RecordType == LogRecordType.Prepare || rec.RecordType == LogRecordType.Stream) &&
+			       || ((rec.RecordType == LogRecordType.Prepare
+			            || rec.RecordType == LogRecordType.Stream
+			            || rec.RecordType == LogRecordType.Stream) &&
```

Comment:
should be EventType

Reply:
ah its needed to get them to appear in the $all stream (for consistency with the stream created records)

Reply:
just for clarification, `|| rec.RecordType == LogRecordType.EventType` needs to be added to the conditions right?

Reply:
yep